NuGet.config

```diff
@@ -13,7 +13,7 @@
     <!--  Begin: Package sources from dotnet-aspnetcore -->
     <!--  End: Package sources from dotnet-aspnetcore -->
     <!--  Begin: Package sources from dotnet-runtime -->
-    <add key="darc-pub-dotnet-runtime-3ba4b3f" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-3ba4b3f4/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-runtime-10b1751" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-10b17515/nuget/v3/index.json" />
```

Comment:
@akoeplinger @steveisok this feed change causes this problem:

> error NU1101: Unable to find package System.Runtime.InteropServices.NFloat.Internal. No packages exist with this id in source(s): darc-pub-dotnet-emsdk-ee0a97a, darc-pub-dotnet-emsdk-ee0a97a-1, darc-pub-dotnet-runtime-10b1751, Dotnet arcade, dotnet-public, dotnet-tools, dotnet6, macios-dependencies, xamarin-impl

Adding back the old feed works around it: https://gist.github.com/rolfbjarne/9ebcc4da595b1e25645c00ced9ecc085, but is that the right (temporary) fix? Or should the new runtime feed also have the System.Runtime.InteropServices.NFloat.Internal package?

Reply:
@rolfbjarne we added that package to the dotnet-public feed just now so you should be able to remove the old feed again.